### PR TITLE
Update banks.coffee

### PR DESCRIPTION
**Pull request by ZeHiro**
Corrected

https://github.com/bnjbvr/kresus/issues/69

### CODE REVIEW

**Comment by ZeHiro** [comment]
Needs to be checked. Only affects banks.coffe. This is first test  for pull request, so I don't get all subtility of pull request.

File: server/controllers/banks.coffee [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports.index = (req, res) ->
         if err?
             h.sendErr res, "when retrieving banks: #{err}"
             return
-        res.send 200, banks
+        res.satus(200).send(banks)
```
Comment (bnjbvr) [inline server/controllers/banks.coffee]: s/satus/status

File: server/controllers/banks.coffee [hunk 2]
```diff
@@ -88,5 +88,5 @@ module.exports.destroy = (req, res) ->
             if err?
                 h.sendErr res, "when deleting access: #{err}"
                 return
-            res.send 204, success: true
+            res.send(204).body(success: true)
```
Comment (bnjbvr) [inline server/controllers/banks.coffee]: Shouldn't it be this? (not sure if it is allowed or not)

```
res.status(204).send success: true
```

**Comment by ZeHiro** [comment]
Yes,, I was to fast.

**Comment by bnjbvr** [comment]
Thank you for the PR, I was planning to do this in some far away future :) Please feel free to address comments and carry on :) If by chance you can get your PR submitted to push into `incoming` rather than `master`, I'd appreciate it, as this is the branch I'm using for unstable development / testing.

**Comment by ZeHiro** [comment]
Ok. I close it, and resubmit one

**Pull request closed without merging**